models/stg_hubspot__ticket.sql

```diff
@@ -20,6 +20,16 @@ with base as (
 ), fields as (
 
     select
+{% if var('hubspot__pass_through_all_columns', false) %}
+        -- just pass everything through
+        {{ 
+            fivetran_utils.remove_prefix_from_columns(
+                columns=adapter.get_columns_in_relation(ref('stg_hubspot__ticket_tmp')), 
+                prefix='property_') 
+        }}
+    from base
+
+{% else %}
```

Comment:
Why do we not exclude ['id'] here and rename it for both instances? It looks like we did this for `company` and `contact` and not for `deal` because deal did not need to be renamed. However, I think ticket should be renamed for consistency.